src/memory_service/fam_memory_service_direct.cpp

```diff
@@ -133,12 +133,19 @@ Fam_Memory_Service_Direct::Fam_Memory_Service_Direct(uint64_t memserver_id,
 
     allocator =
         new Memserver_Allocator(num_delayed_free_Threads, fam_path.c_str());
-    fam_backup_path = (char *)strdup(config_options["fam_backup_path"].c_str());
+    std::string backup_location = config_options["fam_backup_path"];
+    std::size_t found = backup_location.find_last_of("/");
```

Comment:
I did not understand this part of the code.

Reply:
This piece of code was initially added as stat of backup path fails if given backup is a file and not a folder. This code change has been reverted and the existence of fam_backup_path is done in fam_backup/fam_restore/fam_delete_backup APIs